packages/modules/manage-versions/src/components/CreateUpdateVersion/CreateVersionModal/CreateVersionModal.scss

```diff
@@ -2,21 +2,18 @@
  * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
  * See LICENSE.md in the project root for license terms and full copyright notice.
  *--------------------------------------------------------------------------------------------*/
-@import "~@itwin/itwinui-css/scss/variables";
 
 .iac-additional-info {
   display: flex;
   justify-content: space-between;
-  padding: $iui-baseline $iui-sm;
-  @include themed {
-    background-color: t(iui-color-background-2);
-  }
+  padding: var(--iui-size-s);
+  background-color: var(--iui-color-background-zebra);
```

Comment:
why zebra?
It says different one

<img width="618" alt="image" src="https://github.com/iTwin/admin-components-react/assets/81580355/0c9f5fbf-d5f2-4779-9008-21e097a47de6">

Reply:
updated